processes/release_processes/firecloud-develop.dot

```diff
@@ -52,7 +54,8 @@ digraph {
 
   release_cromwell -> dspjenkins_PR
 
-  fcdev_pr -> confirm_cromwell_v
+  fcdev_pr -> helmfile_pr
+  helmfile_pr -> confirm_cromwell_v
```

Comment:
TOLish the existing "confirm_cromwell_v" "check the Fiab version" protocol wouldn't give me a whole lot of warm-and-fuzzy about the correctness of helmfile updates. Also the Fiab bit may be going away starting tomorrow anyway as the runners are folded into k8s.